index.go

```diff
@@ -95,7 +95,7 @@ func (i *Index) Options() IndexOptions {
 }
 
 func (i *Index) options() IndexOptions {
-	return IndexOptions{}
+	return IndexOptions{i.keys}
```

Comment:
should be `IndexOptions{Keys: i.keys}` no?

Reply:
Sure, no big reason not to use the field labels. I think the linter only complains when they are cross-package, but I agree they are good to use. Pushed another commit that adds them.

Reply:
ah, I thought it complained all the time - anywho